### PR TITLE
Restart ccd service only on bootstrap

### PR DESCRIPTION
**Pull request by vpnachev**
**How to categorize this PR?**

/kind enhancement
/priority normal
/os garden-linux

**What this PR does / why we need it**:
Enhance the template to restart the cloud-config-downloader service only in the bootstrap config.
Vendor gardener/gardener@v1.15.5
Updates golang
Updates alpine

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other developer
Alpine has been updated to 3.12.3.
```

```other developer
Golang has been updated to 1.15.7
```

```other developer
Vendor gardener/gardener@v1.15.5
```

```other operator
The `cloud-config-downloader` systemd service is enabled and restarted only in bootstrap `OperatingSystemConfigs`.
```

### CODE REVIEW

File: pkg/generator/templates/cloud-init.gardenlinux.template [hunk 1]
```diff
@@ -74,6 +74,9 @@ fi
 systemctl enable docker && systemctl restart docker
 {{- end }}
 
+{{- $bootstrap := .Bootstrap -}}
 {{- range $_, $unit := .Units }}
+{{- if or (ne $unit.Name "cloud-config-downloader.service") ($bootstrap) }}
 systemctl enable '{{ $unit.Name }}' && systemctl restart '{{ $unit.Name }}'
 {{- end }}
+{{- end }}
```
Comment (rfranzke) [inline pkg/generator/templates/cloud-init.gardenlinux.template:82]: Hm, out of curiosity, does this not work? 

```suggestion
{{- range $_, $unit := .Units }}
{{- if or (ne $unit.Name "cloud-config-downloader.service") ($.Bootstrap) }}
systemctl enable '{{ $unit.Name }}' && systemctl restart '{{ $unit.Name }}'
{{- end }}
{{- end }}
```

**Review by rfranzke — COMMENTED** [review]
/assign

File: pkg/generator/templates/cloud-init.gardenlinux.template (hunk 1 above)
Comment (vpnachev) [reply pkg/generator/templates/cloud-init.gardenlinux.template]: It works.

Comment (rfranzke) [reply pkg/generator/templates/cloud-init.gardenlinux.template]: Cool. Maybe you want to do the same in the other PRs

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**